### PR TITLE
Fix #3774 to add tests for openApi oneOf, allOf and anyOf with annota…

### LINKED ISSUE #3774: [Camel 3.17.0] openApi oneOf, allOf and anyOf with annotation @Schema
Upstream issue: https://issues.apache.org/jira/browse/CAMEL-18040

We need to add a relevant case test for checking the openApi generated result in camel-quarkus.

### PR DESCRIPTION
**Pull request by zhfeng**
…tion @Schema

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: integration-tests/openapi-java/src/main/java/org/apache/camel/quarkus/component/openapijava/it/model/AllOfFormWrapper.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.openapijava.it.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
```
Comment (aldettinger) [inline integration-tests/openapi-java/src/main/java/org/apache/camel/quarkus/component/openapijava/it/model/AllOfFormWrapper.java:22]: So, it's user responsibility to register all model classes for reflection ? If that's the case, would it make sense to document it ?

This is not related to this PR anyway.

**Review by aldettinger — APPROVED** [review]
This PR looks good to me.

File: integration-tests/openapi-java/src/main/java/org/apache/camel/quarkus/component/openapijava/it/model/AllOfFormWrapper.java (hunk 1 above)
Comment (zhfeng) [reply integration-tests/openapi-java/src/main/java/org/apache/camel/quarkus/component/openapijava/it/model/AllOfFormWrapper.java:22]: Good point - I will create a follow issue to update the openapi-java document later.
